shapes/Ellipse.hpp

```diff
@@ -7,22 +7,68 @@ class Ellipse final : public Shape
 {
 public:
     /* Constructors */
+
+    /**
+     * Creates an ellipse centered around a point with a given width and height
+     * @param center the center of the ellipse
+     * @param width the width of the ellipse
+     * @param height the height of the ellipse
+     */
     Ellipse(const QPoint& center, int width, int height);
+
+    /**
+     * Creates an ellipse centered around an (x,y) coordinate pair with a given width and height
+     * @param x the x-coordinate of the center of the ellipse
+     * @param y the y-coordinate of the center of the ellipse
+     * @param width the width of the ellipse
+     * @param height the height of the ellipse
+     */
     Ellipse(int x, int y, int width, int height);
 
     /* Getters */
+
+    /**
+     * Gets the width of the ellipse
+     * @returns the width of the ellipse
+     */
     int getWidth()  const;
```

Comment:
``ctrl + f `` and type double spaces
```suggestion
    int getWidth() const;
```